tests/drivers/adc/adc_dma/src/test_adc.c

```diff
@@ -37,8 +37,8 @@
 #define SAMPLE_INTERVAL_US (10000U)
 
 #define BUFFER_SIZE 24
-static ZTEST_BMEM int16_t m_sample_buffer[BUFFER_SIZE];
-static ZTEST_BMEM int16_t m_sample_buffer2[2][BUFFER_SIZE];
+static ZTEST_BMEM __aligned(32)  int16_t m_sample_buffer[BUFFER_SIZE];
```

Comment:
Please use the newly added dma alignment device tree property for this rather than hard coding it.

Reply:
@teburd, Thanks I have update with dts alignment